website/static/css/pages/contributor-page.css

```diff
@@ -24,8 +24,15 @@
     background-color: #F5F5F5;
 }
 
-#privateLinkTable .btn-group .btn-mini{
-    padding:5px;
+.link-name {
+    display: inline-block;
+}
+
+#privateLinkTable .btn-group {
+    width: 100%;
+    display: flex;
+    display: -webkit-flex;
+    display: -ms-flex;
```

Comment:
Is this doing anything? Flex display wouldn't work without more css for the items right?